pkg/auth/dockercredentials/helper.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2023 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package dockercredentials
+
+import (
+	"errors"
+
+	"github.com/docker/docker-credential-helpers/credentials"
+)
+
+var ErrNotImplemented = errors.New("not implemented")
+
+type RegistryCredentialsGetter interface {
+	GetRegistryCredentials(host string) (string, string, error)
+}
+
+type OktetoClusterHelper struct {
+	getter RegistryCredentialsGetter
+}
+
+var _ credentials.Helper = (*OktetoClusterHelper)(nil)
```

Comment:
This asserts that `OktetoClusterHelper` in fact satisfies the `credentials.Helper` interface. It's not strictly necessary as it would fail to compile we try to use it or pass it down as such but the error will be less cryptic.